nextflow.config

```diff
@@ -60,8 +60,13 @@ params {
     save_reference             = true
     save_alignment             = true
     sample_ploidy              = 1
+<<<<<<< HEAD
```

Comment:
Check here (and also lines 63,66, and 69 for Git tags, e.g. <<<< HEAD, ====, and >>>> upstream/master.  These are added by Git during merging and we will want to remove them here.